packages/common/package.json

```diff
@@ -17,21 +17,16 @@
   "license": "MIT",
   "dependencies": {
     "lodash-es": "^4.17.21",
-    "pathe": "^1.1.2",
```

Comment:
All path dependencies have moved out of commons and the engines so we don't need pathe anymore